core/src/main/java/tc/oc/pgm/listeners/MatchAnnouncer.java

```diff
@@ -74,17 +77,19 @@ public void onMatchEnd(final MatchFinishEvent event) {
     // broadcast match finish message
     for (MatchPlayer viewer : match.getPlayers()) {
       Component title, subtitle = empty();
-      if (event.getWinner() == null) {
+      Collection<Competitor> winners = event.getWinners();
+      if (winners.isEmpty()) {
         title = translatable("broadcast.gameOver");
       } else {
+        boolean plural = winners.size() > 1 || Iterables.getOnlyElement(winners).isNamePlural();
         title =
             translatable(
-                event.getWinner().isNamePlural()
-                    ? "broadcast.gameOver.teamWinners"
-                    : "broadcast.gameOver.teamWinner",
-                event.getWinner().getName());
+                plural ? "broadcast.gameOver.teamWinners" : "broadcast.gameOver.teamWinner",
+                TextFormatter.list(
+                    winners.stream().map(Named::getName).collect(Collectors.toList()),
+                    NamedTextColor.WHITE));
 
-        if (event.getWinner() == viewer.getParty()) {
+        if (winners.stream().anyMatch(w -> w == viewer.getParty())) {
```

Comment:
```suggestion
        if (winners.contains(viewer.getParty())) {
```
Any reason this can't simply be a contains check?

Reply:
Did it to avoid a cast, do you think its worth it?

Reply:
Probably doesn't make a difference, up to you, but i'd leave a comment if you leave the stream in that it's to avoid an unchecked cast when calling contains